lib/collections/schemas/shops.js

```diff
@@ -122,24 +122,6 @@ export const BrandAsset = new SimpleSchema({
 });
 
 /**
- * Shop Taxes
```

Comment:
No need for this anymore. We trust the owner's cloned shop from the DB and only check(shop) when a shop is passed as an argument. Should be safe for creating shops on the fly - but needs testing (as does creating shop after reset - WIP)